test/evergreen.yml

```diff
@@ -38,30 +38,33 @@ functions:
       remote_file: wiredtiger/${build_variant}/${revision}/artifacts/${dependent_task|compile}_${build_id}.tgz
       bucket: build_external
       extract_to: wiredtiger
-  "fetch artifacts from little-endian" :
+  "fetch artifacts from endian format" :
     - command: s3.get
       params:
         aws_key: ${aws_key}
         aws_secret: ${aws_secret}
-        remote_file: wiredtiger/little-endian/${revision}/artifacts/WT_TEST.tgz
+        remote_file: wiredtiger/${endian_format}/${revision}/artifacts/WT_TEST.tgz
         bucket: build_external
-        local_file: WT_TEST-little-endian.tgz
-    - command: archive.targz_extract
-      params:
-        path: "WT_TEST-little-endian.tgz"
-        destination: "wiredtiger/build_posix/test/format"
-  "fetch artifacts from big-endian" :
+        extract_to: wiredtiger/build_posix/test/format
+  "fetch artifacts from many-coll" :
```

Comment:
It seems that the merge with `wt-8498-evergreen-improvements` was not done correctly.

Reply:
Fixed in e2d5c91